assets/css/style.css

```
@@ -1,133 +1,928 @@
+  line-height: 1;
+  text-transform: uppercase;
+  color: #3c3950;
+  font-weight: 300;
+  text-align: center;
+}
+
+.table__title--after {
+  margin-top: 76px;
+  margin-bottom: 96px;
+  font-size: 50px;
+  letter-spacing: 3px;
+  line-height: 1;
+  text-transform: uppercase;
+  color: #3c3950;
+  font-weight: 800;
+  text-align: center;
+}
+
+.content__paragraph__text {
+  padding: 50px 0;
+  margin: 30px 0;
+  text-align: justify;
+  font-size: 16px;
+  line-height: 25px;
+  color: #5f727f;
+  font-weight: 400;
+}
+
+.content__text__adblock {
+  opacity: 0.851;
+  background-color: #494b58;
+}
+
+.item__list {
+}
+
+div i.fa-arrow-right {
+  color: var(--white-color);
+  line-height: 1;
+  font-size: 40px;
+}
+
+#slider {
+  position: relative;
+}
+
+.icon-arrow-right {
+  position: absolute;
+  top: 50%;
+  width: 72px;
+  height: 72px;
+  right: 0;
+  background-color: #212331;
+  text-align: center;
+  line-height: 72px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transform: translateY(-50%);
+}
+
+.icon-arrow-right:hover {
+  background-color: var(--blue-color);
+}
+
+.icon-arrow-left {
+  position: absolute;
+  top: 50%;
+  width: 72px;
+  height: 72px;
+  left: 0;
+  background-color: #212331;
+  text-align: center;
+  line-height: 72px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transform: translateY(-50%);
+}
+
+div i.fa-arrow-left {
+  color: var(--white-color);
+  line-height: 1;
+  font-size: 40px;
+}
+
+.icon-arrow-left:hover {
+  background-color: var(--blue-color);
+}
+
+.img_word {
+  margin: 212px 0 0 0;
+}
+
+.content__paragraph__adblock {
+  padding: 40px 0;
+  margin: 20px 0;
+  line-height: 1;
+  font-size: 100px;
+  letter-spacing: 5px;
+  text-transform: uppercase;
+  color: #f4f6f7;
+  font-weight: 300;
+  text-align: center;
+  font-family: "Montserrat", sans-serif;
+}
+
+.content__paragraph__text_adblock {
+  text-align: center;
+  padding: 50px 40px;
+  font-size: 24px;
+  line-height: 37px;
+  color: #f4f6f7;
+  font-weight: 300;
+  font-family: "Source Sans Pro";
+  margin: 10px 20px;
+}
+
+.footer__before {
+  background-color: #212331;
+  padding: 134px 0;
+}
+.footer__after {
+  height: 100px;
+  background-color: #181828;
+}
+
+.footer__contact {
+  color: #94a4af;
+}
+
+.footer__paragraph {
+  padding: 46px 0;
+  margin-bottom: 10px;
+  color: #94a4af;
+  line-height: 1.5;
+}
+
+.footer_title {
+  padding: 0 0 15px 0;
+  color: #94a4af;
+  margin: 0 0 30px 0;
+  font-weight: 700;
+  color: #feffff;
+  font-size: 19px;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+}
+
+.footer__col__img_head {
+  margin: 5px 0;
+}
+@media (max-width: 500px) {
+  .slide-text-item {
+    font-size: 20px;
+  }
+}
+
+.footer__contact-address {
+}
+hr {
+  border: 1px dashed #4f576a;
+}
+
+.footer__text {
+  line-height: 1.3;
+  padding: 5px 0;
+}
+
+.footer__contact-list-item {
+  margin: 18px 0;
+  padding: 5px 0;
+}
+
+.footer__item {
+  padding: 0 10px 0 0;
```

```suggestion
  margin-right: 10px;
```
Đổi tên class. và nhớ là khoảng cách giữa 2 phần tử là margin